src/actions/session.ts

```diff
@@ -17,7 +17,15 @@ import {
 } from "@src/constants";
 import type { ActionType, AuthData, ServerInfo } from "@src/types";
 
-type NavigationResult = ActionType<typeof notifyError> | { type: null };
+const AUTH_REDIRECT_RESULT: ActionType<typeof notifyError> = {
+  notification: {
```

Comment:
The brief error users could see when redirecting to the auth provider is due to the methods below returning incomplete action data for notification actions. Providing a simple "Redirection to auth provider..." message seems like the simplest solution.